src/checkBoxEvent.js

```diff
@@ -0,0 +1,13 @@
+export default (t) => {
```

Comment:
Please add `npm install` and `npm start` commands in README so a non-technical user can set it up locally.